st2common/bin/migrations/v1.5/st2-migrate-datastore-to-include-scope-secret.py

```diff
@@ -36,11 +36,10 @@ def migrate_datastore():
         for kvp in key_value_items:
             kvp_id = getattr(kvp, 'id', None)
             secret = getattr(kvp, 'secret', False)
-            encrypted = getattr(kvp, 'encrypted', False)
             scope = getattr(kvp, 'scope', SYSTEM_SCOPE)
             new_kvp_db = KeyValuePairDB(id=kvp_id, name=kvp.name,
                                         expire_timestamp=kvp.expire_timestamp,
-                                        value=kvp.value, secret=secret, encrypted=encrypted,
```

Comment:
It appears I changed the data model but the migration script was never changed. This is an issue now because of "additionalProperties: False" change.